xds/src/main/java/io/grpc/xds/RbacFilter.java

```diff
@@ -177,13 +177,13 @@ private ServerInterceptor generateAuthorizationInterceptor(AuthConfig config) {
                 final ServerCall<ReqT, RespT> call,
                 final Metadata headers, ServerCallHandler<ReqT, RespT> next) {
           AuthDecision authResult = authEngine.evaluate(headers, call);
-          if (logger.isLoggable(Level.FINER)) {
-            logger.log(Level.FINER,
+          if (logger.isLoggable(Level.FINE)) {
+            logger.log(Level.FINE,
                 "Authorization result for serverCall {0}: {1}, matching policy: {2}.",
                 new Object[]{call, authResult.decision(), authResult.matchingPolicyName()});
           }
           if (GrpcAuthorizationEngine.Action.DENY.equals(authResult.decision())) {
-            Status status = Status.UNAUTHENTICATED.withDescription(
+            Status status = Status.PERMISSION_DENIED.withDescription(
                     "Access Denied, matching policy: " + authResult.matchingPolicyName());
```

Comment:
Me mentioning FINE in the issue was because this would no longer include the policy name.

Reply:
I briefly searched it, it looks we use FINE more often in our code base, although i don't know which is more proper in which scenario.
Are you suggesting to use FINER level log + including policy name in status, or FINE level log + no include policy name in status, or something else?

Reply:
I'm saying that it seemed like you changed the log level because I said:

> Although that sort of information can be logged at FINE.

But then you ignored the previous sentence:

> It probably isn't appropriate to report which policy failed to the client.